src/resources/elements/primeDesignSystem/pcard/pcard.scss

```diff
@@ -0,0 +1,114 @@
+@import "styles/styles.scss";
+
+.cardContainer {
+  border-radius: 6px;
+  overflow: hidden;
+  & .header {
+    -webkit-text-fill-color: white;
+  }
+}
+
+.card {
+  border-radius: 6px;
+  background-color: $BG02;
+  cursor: pointer;
+
+  padding: 18px;
+
+  [slot="header"] {
+    display: flex;
+    justify-content: space-between;
+    color: $White;
+    margin-bottom: 16px;
+    gap: 6px;
+    >span, >div, >i {
+      width: -webkit-fill-available;
+    }
+
+    .text {
+      font-weight: bold;
+      font-family: "Aeonik";
```

Comment:
Hmmm, I think the design system needs to specify the font families itself as variables in its own _styles.scss